app/konnect/runtime-manager/consumer-groups.md

```diff
@@ -0,0 +1,86 @@
+the plugin accepts
+* Configure a rate limiting policy for each consumer group, overriding the 
+plugin's global configuration
+
+## Set up a consumer group with consumers
+
+In this section, you will create the Bronze tier consumer group with a consumer assigned to the group.
+
+1. In {{site.base_short_name}}, click **Runtime Manager** in the sidebar.
+1. Click the **default** runtime group.
+1. From the menu, click **Consumers**, then click **Add consumer**.
+1. Enter a **Username** and **Custom ID**. For this example, you can use "Amal" for each field.
+1. Click **Save**.
+1. Click the **Consumer Groups** tab.
+1. Click **Add Consumer Group**.
+1. Enter "Bronze" for the consumer group name and select the "Amal" consumer you just created from the drop-down. 
+1. Click **Create**.
+
+## Configure the Rate Limiting Advanced plugin for all consumers
+
+In this section, you will configure the Rate Limiting Advanced plugin to set the rate limit to 5 requests every 30 seconds for all consumers.
+
+1. In {{site.base_short_name}}, click **Runtime Manager** in the sidebar.
+1. Click the **default** runtime group.
+1. From the menu, open **Plugins**, then click **Add plugin**.
+1. Find the **Rate Limiting** plugin, then click **Select**.
+1. Apply the plugin as **Global**, which means the rate limiting applies to all requests, including every service and route in the workspace.
+
+    If you switched it to **Scoped**, the rate limiting would apply the plugin to only one service, route, or consumer.
+
+    By default, the plugin is automatically enabled when the form is submitted.
+    You can also toggle the **This plugin is Enabled** button to configure the plugin without enabling it.
+    For this example, keep the plugin enabled.
+1. Complete only the following fields with the following parameters:
+    1. Config.limit: `5`
+    1. Config.window_size: `30`
+    1. Config.window_type: `Sliding`
+    1. Config.retry_after_jitter_max: `0`
+    1. Config.enforce_consumer_groups: `true` 
+    1. Config.consumer_groups: `Bronze`
+
+    Besides the above fields, there may be others populated with default values. For this example, leave the rest of the fields as they are.
+1. Click **Save**.
+
+## Configure rate limiting for consumer groups
+
+In this section, you will configure the Rate Limiting Advanced plugin to set the rate limit to 6 requests every 30 seconds only for consumers in the Bronze tier.
+
+1. In {{site.base_short_name}}, click **Runtime Manager** in the sidebar.
+1. Click the **default** runtime group.
+1. From the menu, open **Consumers**, then click the **Consumer Groups** tab.
+1. Click the **Bronze** consumer group you just created.
+1. Click **Add Configuration**.
+1. In the dialog, complete only the following fields with the following parameters:
+    1. Window Size: 30
+    1. Window Type: Sliding
+    1. Limit: 6
+    1. Retry After Jitter Max: 0
+1. Click **Save**.
+
```

Comment:
As a tutorial I think this is missing a wrap-up of what was completed and where a user can go from here.

Reply:
@cloudjumpercat Just remove the conclusion heading and I think it will be fine.